fedmsg_migration_tools/verify_missing.py

```diff
@@ -0,0 +1,168 @@
+import json
+import logging
+from datetime import datetime, timedelta
+
+from fedora_messaging import config
+from fedora_messaging.twisted.producer import MessageProducer
```

Comment:
One note about this API, upstream is talking about removing the twisted pika integration[0]. We may want to keep this a private API so it's easy for us to move, or mark it as unstable, or do the work necessary in pika to keep it.

[0] https://github.com/pika/pika/issues/1031

Reply:
Ah, crap :-( I wanted to create a class that other twisted-based tools could use (like the message comparator, but I suspect other tools in our infra too).
Would you recommend using another Twisted-AMQP library like txAMQP? I liked the fact that we could share some stuff with the rest of the pika-based fedora-messaging library, but maybe it's not a reachable goal.
And if no one else is using the pika-twisted connector, I'd rather we don't end up being the only users and maintainers.

Reply:
Yeah...

I believe the only other application really using Twisted is the FMN delivery application and I wrote a hacky little consumer that runs in a thread managed by Twisted to consume. I want to support an asynchronous API, but I don't think it's widely needed (for now). txAMQP looks mostly dead so I think it'd be easy to deal with the Twisted connection in pika than mess with a whole separate library.

What if we kept the Twisted stuff for now (very explicitly marked private) and only used it here? Then we can see how things develop. If the Twisted connection does get dropped, it will presumably come with improvements to the asyncio support. pyzmq already supports that, so moving would be pretty straightforward. I think requiring Python 3 here is reasonable here, even if we can't for fedora-messaging.